trulens/nn/models/tensorflow_v2.py

```diff
@@ -80,6 +82,14 @@ def call(*inputs, **kwargs):
     def B(self):
         return self._B
 
+    def _warn_keras_layers(self, layers):
+        keras_layers = [l for l in layers if 'KerasLayer' in str(type(l))]
+        if (keras_layers):
+            tru_logger.warn('Detected a KerasLayer in the model. This can sometimes create issues during attribution runs or subsequent model calls.\
+            If failures occur: try saving the model, de - activate eager mode with tf.compat.v1.disable_eager_execution(), \
+                set tf.config.run_functions_eagerly(False), then reload the model. '\
```

Comment:
nit: grammar
"If failures occur: try saving the model, deactivating eager mode with `tf.compat.v1.disable_eager_execution()`, setting `tf.config.run_functions_eagerly(False)`, and reloading the model."